llvm/include/llvm/BinaryFormat/DXContainer.h

```diff
@@ -424,6 +424,22 @@ struct ResourceBindInfo : public v0::ResourceBindInfo {
 };
 
 } // namespace v2
+
+namespace v3 {
+struct RuntimeInfo : public v2::RuntimeInfo {
+  uint32_t EntryNameOffset;
+
+  void swapBytes() {
+    v2::RuntimeInfo::swapBytes();
```

Comment:
This v2::RuntimeInfo::swapBytes( ) call is what was missing from my original PR.  This is why the ThreadsX/Y/Z were byte swapped incorrectly on Big Endian machines.